app/models/game.rb

```diff
@@ -4,30 +4,31 @@ class Game < ApplicationRecord
   belongs_to :black_player, class_name: "User", optional: true
   validates :name, presence: true
   after_create :populate_board!
-  
+
   scope :available, -> { where('white_player_id is NULL OR black_player_id is NULL') }
-  
+
   def populate_board!
```

Comment:
I think it over all looks good, but switching the pieces and the colors where there were would break alot of tests and some of us would have to change out logic. I think changing this back would make it good to go